### PR TITLE
admissible colors mention in readme

### PR DESCRIPTION
**Pull request by matanox**
To obviate https://github.com/hamdanal/rich-argparse/issues/117

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,10 +69,10 @@ https://docs.python.org/3/library/argparse.html#formatter-class):
 The default styles used by *rich-argparse* are carefully chosen to work in different light and dark
 themes.
 
-### Customize the colors
+### Customize the font style and colors
```
Comment (hamdanal) [inline README.md:72]: This is not technically correct. The word "style" here is the rich definition of a style (see https://rich.readthedocs.io/en/latest/style.html) which includes colors, and various other attributes like bold, italic, etc. It also includes background colors that have nothing to do with a "font style".
I suggest keeping the old text as it is simple enough for a section title

File: README.md [hunk 2]
```diff
@@ -69,10 +69,10 @@ https://docs.python.org/3/library/argparse.html#formatter-class):
...
 
-You can customize the colors in the output by modifying the `styles` dictionary on the formatter
-class. *rich-argparse* defines the following styles:
+You can customize the font style and colors of the output by modifying the values of the `styles` dictionary on the formatter
+class:
```
Comment (hamdanal) [inline README.md:75]: Same here, "font style" is not correct.

You also removed the sentence that explains what the next example is about: "*rich-argparse* defines the following styles:". This should not be removed.

I suggest using more accurate wording like this:

```md
You can customize the styles of the output by modifying the `styles` dictionary on the formatter
class. You can use any rich style as defined [here](https://rich.readthedocs.io/en/latest/style.html).
*rich-argparse* defines the following styles:
```

The rich styles page has a link to the list of colors at the top, so no need to link to the colors page below.

File: README.md [hunk 3]
```diff
@@ -87,6 +87,8 @@ class. *rich-argparse* defines the following styles:
 }
 ```
 
+Any color supported by [Rich](https://github.com/Textualize/rich) works here ([this list of color names is currently supplied](https://rich.readthedocs.io/en/latest/appendix/colors.html#appendix-colors)).
```
Comment (hamdanal) [inline README.md:90]: This is not useful if you add the link to the style page as suggested above. Please remove.

**Review by hamdanal — CHANGES_REQUESTED** [review]
Thank you for your contribution.
I disagree with some of the changes, please see the comments below.

**Comment by matanox** [comment]
I don't care the least about the phrasing. The point is, how would you suggest people understand which colors can be used if they haven't used Rich before? I only suggest that you make that journey more explicit in any way that you opt.

**Pull request closed without merging**